script/os-autoinst-openvswitch

```diff
@@ -155,12 +164,18 @@ sub _cmd (@args) {
     waitpid($ovs_vsctl_pid, 0);
     my $return_code = $?;
 
+    #warn "### $return_code";
+
```

Comment:
Just a reminder to not forget to remove this at the end of your development work

Reply:
still present